package.json

```diff
@@ -11,13 +11,17 @@
     "mocha": "2.1.0",
     "mocha-phantomjs-core": "2.0.1",
     "promise-polyfill": "6.0.2",
-    "url-search-params": "0.6.1"
+    "url-search-params": "0.6.1",
+    "webpack": "^4.4.1",
+    "webpack-cli": "^2.0.13"
   },
   "files": [
     "LICENSE",
-    "fetch.js"
+    "dist/whatwg-fetch.js"
```

Comment:
If it's desired we could build to `./fetch.js` instead of putting things inside `dist` (and naming it something else). I did it this way as it's more traditional in the community, but otherwise your call.